### DISCUSSION TITLE
Native support for plugins

### DISCUSSION THREAD

**Discussion by VaZark** [sparckles/Robyn#602] [Ideas]
How about native support for runtime plugins? The plugin ecosystem for wordpress is astonishingly huge and is the biggest reason for it's continued success (besides the excellent stewardship). This is also the case for Django which offers an expansive range of installable "django apps".

You can take inspiration from directus which offer something similar by defining the different types of plugins - api routes, internal flows etc.

**Comment by sansyrox** [comment]
Hey @VaZark 👋 

Apologies for the late revert. I missed this discussion. Yes, this is the plan! I agree with your suggestion 😄
